lib/dai-plugin-mcd/contracts/addresses/kovan.json

```diff
@@ -0,0 +1,72 @@
+{    
+    "MULTICALL": "0x4accdab3f3551b90c27419ec27787ee0fdeb5107",
+    "FAUCET": "0xdf31ef13a608728b6fee5d2596ddbb1e0d8bde31",
+    "MCD_DEPLOY": "0x12fb229260acfead5cda087ef9c1f012667d7811",
+    "MCD_GOV": "0x1cc05530723a4fd398b4354ea511a1b0543ba714",
+    "MCD_GOV_GUARD": "0x6db5c4a8a3a013234bdb37580dbc4c19452eb76b",
+    "MCD_ADM": "0x88a23a779cb0550bb6347560ac3daedf0389b098",
+    "VOTE_PROXY_FACTORY": "0xeac98db8e1177d3020f3b3d0e9409a4a396c3981",
+    "MCD_VAT": "0x48f749bd988caafacd7b951abbecc1aa31488690",
+    "MCD_JUG": "0x4e4b50afc59ef5d79f75c4c8731750762fdae24e",
+    "MCD_CAT": "0xc20e4eaf6b53347ad2d895390dd96ba635bbe4c1",
+    "MCD_VOW": "0xb267c22a437637a4672ad61ef69504a8006ccb45",
+    "MCD_JOIN_DAI": "0x4abad252c30380d1d46ce3b5c52cc108b8c778c7",
+    "MCD_FLAP": "0x71264874e428c12e4b4dd5d2a394a3790bfedf02",
+    "MCD_FLOP": "0x64fa7232b1ce6cbef0bd5629823c3c7161dc2e96",
+    "MCD_MOM": "0x793c60586f0be45d9e5c13dfa982dd6d3b5bc9a4",
+    "MCD_MOM_LIB": "0x5c07033c5fca789afb9afe0718de1716c38887af",
+    "MCD_DAI": "0x6503daa2e952d75fba63dba293bce40e3bb9d641",
+    "MCD_DAI_GUARD": "0x3d02a547f864962772f26ec57ab2c85c378dbbdf",
+    "MCD_SPOT": "0x9d762b62221a90302d88d8f4633cb275f558b6e4",
+    "MCD_POT": "0xeda214912cf2bcaf456402a150552bf54e1521e1",
+    "PROXY_ACTIONS": "0xc7948316b2631b0a900595c3713a85a2c49a52b6",
+    "CDP_MANAGER": "0xcba1bbad5fe83cf0bc96028ae3ed8bb98b56986d",
+    "GET_CDPS": "0x2cee0ee17eebfd8eb73fcf795de8c6d8a80111ad",
+    "PROXY_FACTORY": "0xa870e543a88b57da0030509329cb4b2e7c869355",
+    "PROXY_REGISTRY": "0x18af31cf335c1ce14ce3ba53b30f37471419e67a",
+    "WETH": "0xea50804125f432cc2ba32d9341320fdb4e76a309",
+    "ETH": "0xea50804125f432cc2ba32d9341320fdb4e76a309",
+    "PIP_ETH": "0x8C73Ec0fBCdEC6b8C060BC224D94740FD41f3774",
+    "MCD_JOIN_ETH_A": "0x5a992003124c9b663d15556b0b4461e2356d9ddd",
+    "MCD_FLIP_ETH_A": "0x1f952afafee7dbea561211bc4c0c04e4d32b5903",
+    "MCD_JOIN_ETH_B": "0xaf0433bd9f0a87d1c9b445431b7a62b3532adea5",
+    "MCD_FLIP_ETH_B": "0x919a40a1ceca558ab2dec1d7362c5a563aef2e1e",
+    "MCD_JOIN_ETH_C": "0x59e4315413199f212b2649147c0f1f2086e29a61",
+    "MCD_FLIP_ETH_C": "0x4f16415b2ddeb8a611c8ff52cd7d1289c03c6d08",
+    "MCD_JOIN_ETH-A": "0x5a992003124c9b663d15556b0b4461e2356d9ddd",
```

Comment:
i know you're just copying this from the mcd portal, but please remove any lines that have a dash, since we don't use them